evm-abci/src/bin/client.rs

```diff
@@ -1,83 +1,149 @@
 use ethers::prelude::*;
 use evm_abci::types::{Query, QueryResponse};
 use eyre::Result;
+use yansi::{Paint};
+use crate::User::*;
 
-#[tokio::main]
-async fn main() -> Result<()> {
-    // the ABCI port
-    let host = "http://127.0.0.1:3002";
+const ALICE: &str = "0xAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+const BOB: &str = "0xBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB";
+const CHARLIE: &str = "0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC";
 
-    let value = ethers::utils::parse_units(1, 18).unwrap();
-    let alice = "0x2c47Faeff6ED706E9C1D95f51F1800938Cf7c632".parse::<Address>()?;
-    let bob = "0xAAAAAAAAAAAAAABBBBBBBBBBBBBBBB938Cf7c632".parse::<Address>()?;
+#[derive(Copy, Clone)]
+enum User {
+    Alice,
```

Comment:
i'm not dumb right - there's no way to set strings as default enum variant discriminants? if so i could avoid defining the `ALICE`/`BOB`/`CHARLIE` `&str` constants

generally, tho, my approach to mapping users to names/addresses feels hella contrived here...
maybe i should just use a hash map mapping from `Address` -> `name` lol